Backend/index.js

```diff
@@ -641,14 +642,14 @@ app.post('/get-recommended-slots', async (req, res) => {
       const provider = await prisma.serviceProvider.findUnique({ where: { id: providerId } });
 
       if (!user || !provider) {
-          return res.status(404).json({ error: 'User or Service Provider not found' });
+        return res.status(404).json({ error: 'User or Service Provider not found' });
```

Comment:
Are we using a new auto formatter? Just wondering b/c these format changes seem unrelated